lib/r10k/git/rugged/working_repository.rb

```diff
@@ -59,9 +59,14 @@ def clone(remote, opts = {})
   # @param ref [String] The git reference to check out
   # @return [void]
   def checkout(ref)
-    logger.debug1 { "Checking out ref '#{ref}' at #{@path}" }
     sha = resolve(ref)
 
+    if sha
+      logger.debug2 { "Checking out ref '#{ref}' (resolved to SHA '#{sha}') in repository #{@path}" }
```

Comment:
Was this intentionally bumped down. It matches the priority I noticed earlier for fetching so I guess that makes sense.